src/TranslatedApp.tsx

```diff
@@ -0,0 +1,20 @@
+import { useDeskproLatestAppContext } from "@deskpro/app-sdk";
+import type { ReactNode } from "react";
+import { IntlProvider } from "react-intl";
+import locale_en_GB from "../translations/en-GB.json";
+import locale_es_ES from "../translations/es-ES.json";
+
+const locales = {
+  'en-GB': locale_en_GB,
+  'es-ES': locale_es_ES,
+} as const;
+
+export default function TranslatedApp(props: { children: ReactNode }) {
+  const { context } = useDeskproLatestAppContext();
+  const locale = context?.data?.app?.locale ?? 'en-GB';
```

Comment:
What if the context contains a `locale` that isn't listed in the locales array in `manifest.json`?

Reply:
…and we don’t have any translation files for it?